assets/lokomotive-kubernetes/bootkube/resources/charts/lokomotive/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v2
+name: webhook-admission
+description: A Helm chart for Admission webhook for lokomotive
+type: application
+
+version: 0.1.0
+
+appVersion: 1.16.0
```

Comment:
any significance of this appVersion being `1.16.0` ?

Reply:
No, helm automatically created it, will remove it

Reply:
I wouldn't remove it if it's auto-generated.

Reply:
This is auto generated but for a nginx application. I think this should match our lokoctl version.

Reply:
```console
$ helm create foobar
Creating foobar
```

```console
$ grep appVersion foobar/Chart.yaml 
23:appVersion: 1.16.0
```

```console
$ grep nginx foobar/values.yaml 
8:  repository: nginx
46:    # kubernetes.io/ingress.class: nginx
```